proto/common.proto

```diff
@@ -104,8 +104,9 @@ message Run {
   // A string that marks the period of the lhc
   string lhcPeriod = 31;
   // Optional users that stopped or started the run (can also be undefined)
-  optional User user_o2_start = 32;
-  optional User user_o2_stop = 33;
+  // id in this case because Run has a user id in bookkeeping
+  optional int32 userO2Start = 32;
+  optional int32 userO2Stop = 33;
```

Comment:
I think this is not useful, because it will contain the user id, which is specific to bkp. 
I would simply omit this field from gRPC, and we'll add it when one of the client needs it.
(Only for this, not in the run service!)

Reply:
So, do you want me to change it to userIdO2Start then? And I already made functionality for this, so maybe it is good to keep it for now, since you expect it to maybe be needed by a client in the future?

Reply:
No I prefer to simply delete it for now. If at some point we need it, it will not be hard to remove it.
Also: In this proto, you added `id` in User message (in a previous PR https://github.com/AliceO2Group/Bookkeeping/commit/a9641894a0b68e1679a871a02e8317a458e8f956#diff-1747d3070a2311a07389bc9e83309513fbf9f16f7aac7a80988344c38fb98635R127). The thing is, `id` is Bookkeeping internal, and it should not be shared through APIs. I suppose it is not used, and it should be removed.

I understand it is tempting to add code that may be used later, but in the end you have a big codebase with code not used, and the bigger the codebase is, the harder it is to maintain.

Reply:
George told me to make the id optional, that is the only thing I changed, because it is used for log, see the message from George below:
"Indeed, I forgot that we have it for creating a log. In this scenario, the id might be null so you use the existing user message but make the ID optional".

So, I think it is used, I only made it optional for this specific task.

I removed the User fields in common.proto now, but I kept the User message as it is, is that good or do you still want me to remove it with the above information George told me?

Thank you for the explanation, I understand now that the codebase will get increasingly complex.

Reply:
I do not see where it can be used, but you are right, let's keep it from the `User` (it was my fault to put it here in the first place)